schema/test/config/good/spec-example.json

```diff
@@ -275,6 +275,7 @@
             "cpu": {
                 "shares": 1024,
                 "quota": 1000000,
+                "burst": 1000000,
```

Comment:
Period **MUST be no smaller than**(?)

Reply:
> Period MUST be no smaller than burst

Why is this contraint needed? `burst` can be greater than `period`.

Reply:
You're right. Sorry, my bad, I misread the new content on field description above 🙈